### PR TITLE
Simplifying AMP Reward

### PR DESCRIPTION
**Pull request by michael-lutz**
The original AMP reward never really made sense to me 1) because the max term reduces expressiveness of the term and 2) I don't understand why you'd want to add a quadratic transformation on discriminator logits when they're trained to linearly interpolate between -1 and 1.

### CODE REVIEW

**Comment by codekansas** [comment]
nice, lgtm

**Review by codekansas — APPROVED** [review]
